.travis.yml

```diff
@@ -114,17 +142,26 @@ jobs:
         - npm run bootstrap-no-build
         - echo 'BOOTSTRAP FINISHED'
       script: ./travis-ci/travis_wait_new 50 ./travis-ci/run-redeployment-tests.sh
-      after_failure: ./travis-ci/travis_wait_new 50 ./travis-ci/cleanup-integration-tests.sh
+      after_failure: ./travis-ci/travis_wait_new 50 ./travis-ci/unlock-stack.sh
```

Comment:
I changed this because if the nightly integration tests fail, it goes ahead and deletes the stack which is not helpful for debugging the failure, or figuring out what happened at all!